debian/control

```diff
@@ -101,7 +101,8 @@ Recommends: btrfs-tools,
             ceph-mds (= ${binary:Version}),
             librados2 (= ${binary:Version}),
             libradosstriper1 (= ${binary:Version}),
-            librbd1 (= ${binary:Version})
+            librbd1 (= ${binary:Version}),
+            time-daemon
```

Comment:
time-daemon is a virtual package so you need to provide a primary 'real' package as well:

   ntp | time-daemon

That way, its either ntp or another package that Provides: time-daemon if also provided on the apt-get call.